### DISCUSSION TITLE
Rendering of leisure=garden + garden:type=residential + access=private

### DISCUSSION THREAD

**Discussion by Dr-Mx** [organicmaps/organicmaps#7635] [Ideas]
This is probably me going a bit crazy with micro-mapping as of late, however rendering of these residential gardens on Organic Maps can make public parks/gardens become rather indistinct. May I suggest to avoid rendering these or using a much lighter colour?

https://omaps.app/44IAxoezek
https://wiki.openstreetmap.org/wiki/Tag:garden:type=residential

Kind regards.

**Comment by hb0nd** [comment]
agreed 👍️ have transferred to an issue (https://github.com/organicmaps/organicmaps/issues/7636)
